src/test/java/edu/ucsb/cs156/rec/integration/RequestTypeIT.java

```diff
@@ -0,0 +1,57 @@
+package edu.ucsb.cs156.rec.integration;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
```

Comment:
This file appears to contain no tests.   We should not add this to the code base unless/until there are actually some tests in it.